src/ResourceManager/Sql/Commands.Sql/help/AzureRM.Sql.md

```diff
@@ -8,7 +8,7 @@ Locale: en-US
 
 # AzureRM.Sql Module
 ## Description
-This topic displays help topics for the [Azure SQL Database](https://docs.microsoft.com/azure/sql-database/) Cmdlets.
```

Comment:
@akromm that's the correct command. I would assume this is an issue with how `platyPS` parses the markdown (maybe it can't handle links?). I can investigate myself and open an issue in their repository if necessary.